karate-core/src/main/java/com/intuit/karate/driver/DriverOptions.java

```diff
@@ -125,6 +125,10 @@ public String getPreSubmitHash() {
         return preSubmitHash;
     }
 
+    public boolean isRemoteHost() {
+        return options != null && options.get("remoteHost") != null && (Boolean) options.get("remoteHost");
```

Comment:
can't we formalize this like all the others in the constructor which avoids the null checks etc and this should go into the readme for karate-core

Reply:
but if this is so rare maybe the docs don't need updating. I'll merge now anyway and you can consider a delta

Reply:
Yup I'll do some clean up o avoid the null checks. I think it's still worth updating the docs, not sure why more people aren't using sibling or creating a their own farm of chrome's tbh